### PR TITLE
feat: add squared two body momentum function

### PR DESCRIPTION
**Pull request by spflueger**
This adds a squared version of the two body momentum which is easier to use in terms of analytic continuation instead of the complex version.

### CODE REVIEW

File: amplitf/kinematics.py [hunk 1]
```diff
@@ -999,6 +999,20 @@ def zemach_tensor(m2ab, m2ac, m2bc, m2d, m2a, m2b, m2c, spin):
 
 
 
+@atfi.function
+def two_body_momentum_squared(md, ma, mb):
```
Comment (redeboer) [inline amplitf/kinematics.py:1003]: How about reusing this function below in `two_body_momentum`?

Comment (spflueger) [reply amplitf/kinematics.py:1003]: true

File: setup.py [hunk 2]
```diff
@@ -8,7 +8,7 @@ def long_description():
 
 setuptools.setup(
     name="amplitf",
-    version="0.0-alpha1",
+    version="0.0-alpha2",
```
Comment (redeboer) [inline setup.py:11]: Not a semantic version, right?

Comment (redeboer) [reply setup.py:11]: I would also keep these versions as low as possible for now (it's experimental anyway). So I would say `0.0.1-alpha`.

**Pull request merged**